ci/cloudbuild/builds/cmake-install.sh

```diff
@@ -51,6 +51,9 @@ mapfile -t actual_dirs < <(env -C "${INSTALL_PREFIX}" find -type d)
 mapfile -t expected_dirs < <(cat ci/etc/expected_install_directories)
 expected_dirs+=(
   ./include/google/api
+  ./include/google/bigtable
```

Comment:
nit: this change seems unrelated, but I guess you had to do it as part of regenerating all the libraries?

Reply:
Correct.  That is, apart from the generator changes themselves, everything else here was either generated or required by a test because of the regeneration.